api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -75,6 +75,15 @@ message HttpConnectionManager {
     PASS_THROUGH = 2;
   }
 
+  enum SchemeHeaderTransformation {
+    // Overwrite any Scheme header with the contents of scheme
+    OVERWRITE_SCHEME = 0;
+
+    // If no Scheme header is present, append Scheme scheme
```

Comment:
Since we'd always be appending to an empty string should this be called `ADD_SCHEME_IF_ABSENT` instead?